algoliasearch/src/main/java/com/algolia/search/saas/BaseAPIClient.java

```diff
@@ -62,7 +62,9 @@
     private List<String> readHosts;
     private List<String> writeHosts;
 
-    /** HTTP headers that will be sent with every request. */
+    /**
+     * HTTP headers that will be sent with every request.
+     */
```

Comment:
For this commit, it's fine. For later commits, it's best in my opinion to minimize code changes, because:

1. Small PRs are easier to review than large PRs with a lot of noise.
2. Reformatting hinders traceability (like Git blame).

Of course it would be better if consistent coding conventions had been used from the start. But unfortunately it's not the case, so I say it's better to keep the history than to please our eyes.